rs/nns/test_utils/src/common.rs

```diff
@@ -394,3 +406,9 @@ pub fn build_mainnet_governance_wasm() -> Wasm {
     let features = [];
     Project::cargo_bin_maybe_from_env("mainnet-governance-canister", &features)
 }
+
+/// Build Wasm for Index canister for the ICP Ledger
+pub fn build_mainnet_index_wasm() -> Wasm {
+    let features = [];
+    Project::cargo_bin_maybe_from_env("mainnet-index-canister", &features)
```

Comment:
```suggestion
    Project::cargo_bin_maybe_from_env("mainnet-ic-icp-index-canister", &features)
```

Reply:
> It would be good to run sns-testing with both configurations: mainnet and the current version of all the canisters.

Do `ic` releases contain the test version of the governance canister?
We'll need the test version to run real-time scenarios (to avoid waiting 24h for swap to open 😅)

Reply:
That's an excellent question. 

No, test canister versions are of course never deployed to the mainnet. 

But the mainnet version was built from a known Git commit, so we could take the test version of the same commit. 

Since this is not a dependency that we currently have in the mono repo's Bazel configuration, let's not take the d-tour and stick with the current canister versions, for now.

Reply:
The suggested change is probably still needed, though, right?

Reply:
Yes, your suggestion is completely valid 😅